go/install/install_darwin.go

```diff
@@ -1108,3 +1151,102 @@ func InstallLogPath() (string, error) {
 func SystemLogPath() string {
 	return "/Library/Logs/keybase.system.log"
 }
+
+func fallbackPIDFilename(service launchd.Service) string {
+	return filepath.Join(os.TempDir(), "kbfb."+service.Label())
```

Comment:
I'm not sure about using `os.TempDir()` here.  I guess it's safe since macOS is so aggressive about giving you a private /tmp dir, but might it be better to use the CacheDir?

Reply:
I used it in case `CacheDir` has bad permissions.

Reply:
OK, that makes sense.  Looking in my /private/tmp, I have a mysqld.sock there.  I guess this is OK?  It feels a little bit sketchy.  Can it be cleaned out as in Linux?

Reply:
It gets removed in Uninstall.

Reply:
also, it won't be in /private/tmp but in something like `/var/folders/gm/j_mf88bd5nx77vsx6wks8z2w0000gn/T/`

Reply:
oh right, that's good.